### PR TITLE
Front end routing

### PR DESCRIPTION
**Pull request by Kumjami**
## WIP

Issues:  https://github.com/numo-labs/isearch-ui/issues/117 and https://github.com/numo-labs/isearch-ui/issues/98. 

Using [react-router](https://github.com/reactjs/react-router/) we will be able to have some routing logic, browser history and to share links about articles and packages.

### CODE REVIEW

**Comment by Kumjami** [comment]
Current status:
Waiting for fixing `graphql` schema.

**Comment by Kumjami** [comment]
Waiting for https://github.com/numo-labs/lambda-numo-graphql/pull/104

File: lib/slider/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component, PropTypes } from 'react';
 import Slider from 'react-slick';
-import { SliderArrowRight, SliderArrowLeft } from './arrows.js';
+// import { SliderArrowRight, SliderArrowLeft } from './arrows.js';
```
Comment (nikhilaravi) [inline lib/slider/index.js]: these arrows don't seem to be working... It cause an 'invalid' violation in react so I commented them out. Do you know why they are not working?

Comment (Kumjami) [inline lib/slider/index.js]: @nikhilaravi No Idea, they were working, maybe something was lost in a merge. I will take a look :+1:

**Pull request merged**